src/views/GovernanceTaskList/index.tsx

```diff
@@ -95,15 +94,14 @@ const businessCaseLinkComponent = ({
   businessCaseStatus,
   history
 }: businessCaseLinkComponentProps) => {
-  const path = chooseBusinessCasePath(businessCaseStatus, businessCase.id);
```

Comment:
This was the one place `chooseBusinessCasePath` was invoked.

The switch statement in `chooseBusinessCasePath` is the same as the switch statement that is in this function, so we can just put the paths directly into the button/link instead of doing all that logic.